### PR TITLE
Fixing test case and mode function for column_stats

### PR DESCRIPTION
**Pull request by Rowansiv**

### CODE REVIEW

File: tests/test_column_stats.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import pandas as pd
 import numpy as np
 import statistics
+from EDAhelper.column_stats import column_stats
 
 data = pd.read_csv('https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv')
```
Comment (suuuuperNOVA) [inline tests/test_column_stats.py:6]: iris =

Comment (suuuuperNOVA) [inline tests/test_column_stats.py:6]:
```suggestion
iris = pd.read_csv('https://raw.githubusercontent.com/mwaskom/seaborn-data/master/iris.csv')
```

File: tests/test_column_stats.py [hunk 2]
```diff
@@ -16,20 +17,26 @@ def variance(data):
...
-assert column_stats(iris, cols)[1][cols[1]].index[0] == cols[0]
-assert column_stats(iris, cols)[1][cols[1]].index[1] == cols[1]
-assert column_stats(iris, cols)[2][cols[1]].index[0] == cols[0]
-assert column_stats(iris, cols)[2][cols[1]].index[1] == cols[1]
-assert column_stats(iris, cols)[0]['Column'][0] == cols[0]
-assert column_stats(iris, cols)[0]['Mean'][0] == round(iris['petal_width'].sum()/iris['petal_width'].count(), 3)
-assert column_stats(iris, cols)[0]['Var'][0] - 0.5 <= variance(iris['petal_width']) <= column_stats(iris, ('petal_width', 'petal_length'))[0]['Var'][0] + 0.5
-assert column_stats(iris, cols)[1]['petal_width'][0] == 1
-assert column_stats(iris, cols)[1]['petal_length'][1] == 1
+def column_args_in_outputs:
```
Comment (suuuuperNOVA) [inline tests/test_column_stats.py]:
```suggestion
def column_args_in_outputs(iris):
```

File: tests/test_column_stats.py [hunk 3]
```diff
@@ -16,20 +17,26 @@ def variance(data):
...
+    assert column_stats(iris, cols)[1][cols[1]].index[0] == cols[0], 'Column names do not match'
+    assert column_stats(iris, cols)[1][cols[1]].index[1] == cols[1], 'Column names do not match'
+    assert column_stats(iris, cols)[2][cols[1]].index[0] == cols[0], 'Column names do not match'
+    assert column_stats(iris, cols)[2][cols[1]].index[1] == cols[1], 'Column names do not match'
+    
+
+def column_values_calculated:
```
Comment (suuuuperNOVA) [inline tests/test_column_stats.py]:
```suggestion
def column_values_calculated(iris):
```

File: tests/test_column_stats.py [hunk 4]
```diff
@@ -16,20 +17,26 @@ def variance(data):
...
+    assert column_stats(iris, cols)[1]['petal_length'][1] == 1, 'Covariance calculated incorrectly'
 
 '''
 Checking all dataframes have same number of rows as number of specified columns in function
 '''
-assert len(column_stats(iris, cols)[0].columns) == 9
-assert len(cols) == len(column_stats(iris, cols)[0])
-assert len(cols) == len(column_stats(iris, cols)[1])
-assert len(cols) == len(column_stats(iris, cols)[2])
+def num_cols:
```
Comment (suuuuperNOVA) [inline tests/test_column_stats.py]:
```suggestion
def num_cols(iris):
```

**Review by suuuuperNOVA — CHANGES_REQUESTED** [review]
Check the suggestions

File: tests/test_column_stats.py (hunk 2 above)
Comment (jennifer-hoang) [inline tests/test_column_stats.py]:
```suggestion
def test_column_args_in_outputs:
```

**Pull request merged**